src/status_im/chat/events.cljs

```diff
@@ -120,38 +110,27 @@
                                  :show-emoji?       false
                                  :bottom-info       details})))
 
+(def ^:private index-messages (partial into {} (map (juxt :message-id identity))))
```

Comment:
We don't use `partial` much in the codebase.

Reply:
What's wrong on partial ?

Reply:
Nothing but it's a different mental model (like `comp`) that is not used in this codebase. Adds overhead to read code as it's not a common pattern in the project.

Reply:
How exactly does `partial` add overhead?

Reply:
`^:private` literally adds nothing here, so i wouldn't use it